consensus/misc/kip71.go

```diff
@@ -23,10 +23,29 @@ func VerifyMagmaHeader(config *params.ChainConfig, parentHeader, header *types.H
 	return nil
 }
 
+func makeEvenByDown(baseFee *big.Int) *big.Int {
+	if baseFee.Bit(0) != 0 {
+		baseFee.Rsh(baseFee, 1)
+		baseFee.Lsh(baseFee, 1)
+		return baseFee
```

Comment:
```suggestion
```

Reply:
It needs return statement to use something like `return makeEvenByUp(baseFee)` in call sites.

Reply:
ahh, ok thanks!